shuup/core/models/_order_lines.py

```diff
@@ -102,6 +113,10 @@ class AbstractOrderLine(MoneyPropped, models.Model, Priceful):
 
     created_on = models.DateTimeField(auto_now_add=True, editable=False, db_index=True, verbose_name=_('created on'))
     modified_on = models.DateTimeField(auto_now=True, editable=False, db_index=True, verbose_name=_('modified on'))
+    line_behaviour = EnumIntegerField(
```

Comment:
This is not needed in OrderLine model, once the order is created, this shouldn't be useful anymore